mvp-spec/prebid-implementation.md

```diff
@@ -0,0 +1,691 @@
+```
+<!--partial-end-->
+
+## Implementation Details
+
+PAF doesn't standardize the way of offering inventory because 
+there are already mechanisms sush as OpenRTB for these cases.
+However to offer PAF the Publisher must implement additional features.
+
+To understand the steps, it is important to overview how to generate the data and
+the relationships between them:
+* A Publisher offers *multiple* placements - one for each for 
+Addressable Contents - via Prebid.js.
+* A Publisher Server generates a Seed for each PAF Data transaction
+* Prebid.js sends a Request for opportunites and *one* Transaction Request
+* A Supplier generates and sends *one* Transaction Response per Impression Opportunity
+
+
+### Prebid.js userId module
+
+Prebid.js already offers identity submodules and a PAF sub module will be added.
+This module will be very straight-forward and will simply fetch the PAF data from the
+window using the [PAF frontend](https://github.com/criteo/paf-mvp-implementation/tree/main/paf-mvp-frontend). The module will call
+`PAF.getIdsAndPreferences()` to get the PAF Data. It will next call `PAF.generateSeeds(1)`
+to retrieve a transmission seed. These data will be combined and offered to all
+configured bidders in the transmission request format:
+
+<!--partial-begin { "files": [ "transmission-requests.json" ], "block": "json" } -->
+<!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
+```json
+{
+    "data": {
+        "identifiers": [
+            {
+                "version": 0,
+                "type": "prebid_id",
+                "value": "7435313e-caee-4889-8ad7-0acd0114ae3c",
+                "source": {
+                    "domain": "operator0.com",
+                    "timestamp": 1639580000,
+                    "signature": "12345_signature"
+                }
+            }
+        ],
+        "preferences": {
+            "version": 0,
+            "data": { 
+                "opt_in": true 
+            },
+            "source": {
+                "domain": "cmp1.com",
+                "timestamp": 1639581000,
+                "signature": "12345_signature"
+            }
+        }
+    },
+    "transmissions": [
+        {
+            "version": 0,
+            "seed": {
+                "version": 0,
+                "transaction_id": "a0651946-0f5b-482b-8cfc-eab3644d2743",
+                "publisher": "publisher.com",
+                "source": {
+                    "domain": "publisher.com",
+                    "timestamp": 1639582000,
+                    "signature": "12345_signature"
+                }
+            },
+            "source": {
+                "domain": "dsp1.com",
+                "timestamp": 1639581000,
+                "signature": "12345_signature"
+            },
+            "parents": []
+        },
+        {
+            "version": 0,
+            "seed": {
+                "version": 0,
+                "transaction_id": "a0651946-0f5b-482b-8cfc-eab3644d2743",
```

Comment:
This  transaction_id is the same as the one above.
If the intent is to show how it works with several seeds, it should be a different transaction_id.
Is the intent is to show how it works with a single seed, this transmission should be removed.

Reply:
This is an existing transmission created by @RomainLofaso , maybe he can give feedback.